services/keywords/main.py

```diff
@@ -0,0 +1,13 @@
+import utils
+
+def get_keywords_service(request):
```

Comment:
Please add tests, you can fake out an incoming flask request by doing:

```
def make_flask_request(json):
    return flask.Request.from_values(content_type="application/json", data=json)
```

You'll probably want to use a test library like `unittest.mock` to help mock out `gcloud_syntax_extraction`: https://fgimian.github.io/blog/2014/04/10/using-the-python-mock-library-to-fake-regular-functions-during-tests/

Feel free to use a different `pytest` one though.